src/app/campaign/flight/flight.component.html

```diff
@@ -35,4 +35,17 @@
       </mat-select>
     </mat-form-field>
   </form>
+  <div class="flight-controls-container">
+    <button
+      (click)="toggleDeleteFlight()"
+      mat-fab
+      [color]="softDeleted ? 'primary' : 'warn'"
+      [attr.aria-label]="softDeleted ? 'Restore flight' : 'Delete flight'"
+    >
+      <mat-icon>{{ softDeleted ? 'restore_from_trash' : 'delete' }}</mat-icon>
```

Comment:
we're actively using several icon implementations, the prx-icon in the styleguide built by Rick, the global icon styles, and now material icons, yay more tech debt

Reply:
🙁 ... yay. Again hoping for clarity after #57.